community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/NodeFirstRelationshipProcessor.java

```diff
@@ -52,15 +52,13 @@ public boolean process( NodeRecord node )
     {
         long nodeId = node.getId();
         long firstRel = nodeRelationshipLink.getFirstRel( nodeId, this );
-        if ( firstRel == -1 )
+        if ( firstRel != -1 )
         {
-            return false;
```

Comment:
That's due to the mode removal in `BatchingPageCache`. We always have to update the writer with all records since it never reads from the store. Previously the rare holes in the stores could be skipped, not anymore.